src/nvim/drawscreen.c

```diff
@@ -264,51 +266,39 @@ void screen_resize(int width, int height)
   p_lines = Rows;
   p_columns = Columns;
 
-  // was invoked recursively from a VimResized autocmd, handled as a loop below
-  if (resizing_autocmd) {
-    return;
```

Comment:
Whenever `resizing_autocmd` is true, `resizing_screen` is also true, so this function should have already returned above, and this check is not needed.